README.md

```diff
@@ -26,64 +29,70 @@ Basic Concept
 Getting Started
 ---------------
 
-- You need Python3. We tested primarily with **python 3.5**
-- This was tested with a recent Ubuntu based Linux.
-
+- You need Python3.6 and above for installing tanner.
+- This was tested with a recent Ubuntu-based Linux.
 
-### Setup Redis
+### Steps to install TANNER
 
+#### Step 1: Setup Redis
 
 1. Install the Redis: ``sudo apt-get install redis-server``
-2. Start it on ``localhost`` with default ``port``: ``redis-server``
-
-### Setup PHP Sandbox
+2. Run ``redis-server`` (to start it on `localhost` with default `port`)
 
+#### Step 2: Setup PHP Sandbox
 
-1. For PHP Sandbox setup, see sandbox [manual](https://github.com/mushorg/phpox)
-2. In PHP Sandbox directory, run sandbox: ``sudo python3 sandbox.py``
+1. Clone the repo using:  ``git clone https://github.com/mushorg/phpox.git`` (preferably in your home directory).
+2. Go to PHP Sandbox directory: ``cd phpox``
+2. Run sandbox: ``sudo python3 sandbox.py``
 
-### Setup Docker
+For more help on PHP Sandbox setup, see sandbox [manual](https://github.com/mushorg/phpox)
 
+#### Step 3: Setup Docker
 
-1. Install [docker](https://docs.docker.com/engine/installation/linux/ubuntu/)
-2. Pull the required image to use [default : ``busybox:latest``]
```

Comment:
Why you've deleted this from REDME?

Reply:
I have replaced the 1st step with the direct command, which will install the docker (which is given in the link), and shifted the link below if needed for troubleshooting.

Regarding the second point, I don't know what exactly it meant, and I was able to setup tanner on docker just using those commands which I have listed, hence deleted it.

Reply:
I was talking about the second point :) I was trying to figure out why we had it explicitly, while host image is defined in config and managed by our docker_helper :)

Reply:
Yeah exactly its redundant